air-script/tests/aux_trace/aux_trace.air

```diff
@@ -7,6 +7,9 @@ trace_columns:
 public_inputs:
     stack_inputs: [16]
 
+random_values:
+    rand: [2]
+
```

Comment:
Let's remove this from here and add a specific `random_values.air` test that tests both types of random values accessors (`$rand` and named random values)